packages/context/README.md

```diff
@@ -199,6 +199,8 @@ absolute URL.
 
 ### SignalR Connection
 
+@depreacated
```

Comment:
Typo `deprecated`.

Also, is this some markdown syntax, or why the reason for `@`?

Reply:
No it is not